package.json

```diff
@@ -7,20 +7,32 @@
     "url": "react-bootstrap/react-bootstrap"
   },
   "homepage": "https://react-bootstrap.github.io/",
+  "main": "lib/index.js",
+  "jsnext:main": "es/index.js",
+  "module": "es/index.js",
   "scripts": {
     "bootstrap-docs": "npm run build lib && cd www && yarn install && cd ..",
     "build": "node tools/build.js",
     "build-docs": "npm run bootstrap-docs && npm -C www run build",
-    "lint": "eslint www/src src test tools *.js",
+    "ci-lint": "eslint --rule 'prettier/prettier: 2'",
```

Comment:
i guess this is ok

Reply:
yeah idk, maybe we find something a bit less awkward but it seems to work well enough for now anyway